configure

```diff
@@ -3059,16 +3059,21 @@ EOF
 		append_var CXXFLAGS "-mlong-calls"
 		append_var CXXFLAGS "-mword-relocations"
 		append_var CXXFLAGS "-fomit-frame-pointer"
-		#use link time optimization to further reduce exe size
-		append_var CXXFLAGS "-flto"
-		append_var LDFLAGS "-flto=jobserver"
 		#ensure verbose output during linking to prevent buildbot kills after 1200 seconds
 		if test "$_verbose_build" = yes ; then
 			append_var LDFLAGS "-Wl,-v --verbose"
 		fi
-		#use linker dead code elimination to further reduce exe size
-		append_var CXXFLAGS "-ffunction-sections -fdata-sections"
-		append_var LDFLAGS "-Wl,--gc-sections"
+		if test "$_dynamic_modules" = yes ; then
+			_detection_features_static=no
+			_plugins_default=dynamic
+		else
+			#use link time optimization to further reduce exe size
+			append_var CXXFLAGS "-flto"
+			append_var LDFLAGS "-flto=jobserver"
+			#use linker dead code elimination to further reduce exe size
+			append_var CXXFLAGS "-ffunction-sections -fdata-sections"
+			append_var LDFLAGS "-Wl,--gc-sections"
```

Comment:
I'd recommend moving this further down, since this flag has previously interfered with library detection.

Reply:
I don't understand what you mean.
This is used when the build is fully static and was already at this place.
Indeed, the garbage collection of sections must not be used in plugins build.

Reply:
The DS, 3DS and RISC OS ports add this near the end of the configure script to prevent them from interfering when attempting to detect the availability of certain symbols. It might be a good idea to do the same thing for PS Vita as well, since this is being changed anyway.

Reply:
Ah yes, you proposed an improvement. :)
Done.